repl/src/main/scala/ammonite/repl/interp/Compiler.scala

```diff
@@ -249,13 +250,15 @@ object Compiler{
      * Compiles a blob of bytes and spits of a list of classfiles
      */
     def compile(src: Array[Byte],
-                printer: Printer): Output = {
+                printer: Printer,
+                impLen: Int): Output = {
```

Comment:
Can we call this `importsLen0` instead of `impLen` so it's more obvious that it's related to `this.importsLen`? And let's update the comment above to briefly mention why we need to pass it in (so we can properly update the positions of the generated ASTs for better errors)